cmd/flux/debug_helmrelease.go

```diff
@@ -109,5 +130,19 @@ func debugHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		rootCmd.Print(string(values))
 	}
 
+	if debugHelmReleaseArgs.showHistory {
+		if hr.Status.History == nil || len(hr.Status.History) == 0 {
```

Comment:
```suggestion
		if len(hr.Status.History) == 0 {
```

`len(arr)` returns 0 if `arr == nil`